test/database/service/BookingServiceTest.java

```diff
@@ -59,4 +59,71 @@ public void testSaveGuest() {
 
 		verify(guestDataDAO).save(any(GuestDataDTO.class));
 	}
+	
+	@Test
+	public void testLoadBookingList() {
+		BookingDataDTO bookingDataDTO=new BookingDataDTO(LocalDateTime.now(),LocalDateTime.of(2023,12,12,10,30),PaymentMethodDTO.CASH,new BigDecimal("6000"));
+		BookingDataDTO bookingDataDTO1=new BookingDataDTO(LocalDateTime.now(),LocalDateTime.of(2024,12,12,10,30),PaymentMethodDTO.CASH,new BigDecimal("7000"));
+		BookingDataDTO bookingDataDTO2=new BookingDataDTO(LocalDateTime.now(),LocalDateTime.of(2025,12,12,10,30),PaymentMethodDTO.CASH,new BigDecimal("8000"));
+		
+		List<BookingDataDTO>listBookingDataDTO=new ArrayList<>();
+		
+		listBookingDataDTO.add(bookingDataDTO);
+		listBookingDataDTO.add(bookingDataDTO1);
+		listBookingDataDTO.add(bookingDataDTO2);
+		
+		Mockito.doReturn(listBookingDataDTO).when(bookingDataDAO).searchBookingList();
+		
+		List<BookingDataDTO> loadBookingList = bookingService.loadBookingList();
+		
+		verify(bookingDataDAO).searchBookingList();
+		
```

Comment:
Please validate list `loadBookingList` is equal to `listBookingDataDTO`, it must be equal.